python_modules/dagster/dagster/core/instance/__init__.py

```diff
@@ -1107,11 +1108,44 @@ def wipe_assets(self, asset_keys):
 
     # event subscriptions
 
-    def get_event_log_handler(self):
+    def _get_yaml_python_handlers(self):
+        if self._settings:
+            logging_config = self.get_settings("python_logs").get("dagster_handler_config", {})
+
+            # raise exception here if loggers provided in logging_config
+            if "loggers" in logging_config:
+                raise Exception("Loggers should not be specified in the configuration file.")
+
+            # Handlers can only be retrieved from dictConfig configuration if they are attached
+            # to a logger. We add a dummy logger to the configuration that allows us to access user
+            # defined handlers.
+            handler_names = logging_config.get("handlers", {}).keys()
+
+            dummy_logger_name = "dummy_logger"
```

Comment:
maybe make this something like "__dagster_dummy_logger__" just to be extra safe? I think in the future we might be able to get away with making this just straight up "dagster", and allowing people to add handlers in python as well as the yaml config, but I think we can just keep things simple for now.